StPetersburg/InEtn210325.xml

```diff
@@ -133,11 +133,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     <term key="Translation"/>
                 </keywords>
             </textClass>
-            <langUsage><language ident="en">English</language><language ident="am">Amharic</language></langUsage>
```

Comment:
Amharic is used in the addition and should also be kept here, in addition to Geez

Reply:
corrected